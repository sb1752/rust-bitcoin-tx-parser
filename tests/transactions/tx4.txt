02000000021a6d5a57b061e719f20817a075a6909b4b77ee1829ca72c7cf0ecaee61fd9f68020000008a4730440220318629d8d11c324f9a4369f1e5e63d30bbfc3f5d0b3a5f6ffc236472a2e7554102201911518d139a7c4f5a56104c72869b651bb1c61d438ba09a0e4136941f5a08190141047146f0e0fcb3139947cf0beb870fe251930ca10d4545793d31033e801b5219abf56c11a3cf3406ca590e4c14b0dab749d20862b3adc4709153c280c2a78be10cffffffffb7bc6adc6bb335f986d2623c75145208a97ea857c9efc8f80765932977d4cdb7030000008a47304402202ede02a79522a6d64d5088140d7f1abc412c26d9ea093c2fbb321b8f583f279e02207bb12a722558103009abda6f995e857cf04c98c6e8bd01ff7010267bd8a4bd1d0141047146f0e0fcb3139947cf0beb870fe251930ca10d4545793d31033e801b5219abf56c11a3cf3406ca590e4c14b0dab749d20862b3adc4709153c280c2a78be10cffffffff03f07e0e000000000017a91401a2723c1e23a2bfd241c5ded1973e0d1c9ce2488710c9b66f000000001976a9149a803a4f5e37b420a904b1dfcf96c70a7300082d88acb309be99000000001976a91443849383122ebb8a28268a89700c9f723663b5b888ac00000000