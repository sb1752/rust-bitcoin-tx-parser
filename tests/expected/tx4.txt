ParsedTransaction {
    version: 2,
    input_count: 2,
    inputs: [
        Input {
            txid: "689ffd61eeca0ecfc772ca2918ee774b9b90a675a01708f219e761b0575a6d1a",
            vout: 2,
            script_sig: "4730440220318629d8d11c324f9a4369f1e5e63d30bbfc3f5d0b3a5f6ffc236472a2e7554102201911518d139a7c4f5a56104c72869b651bb1c61d438ba09a0e4136941f5a08190141047146f0e0fcb3139947cf0beb870fe251930ca10d4545793d31033e801b5219abf56c11a3cf3406ca590e4c14b0dab749d20862b3adc4709153c280c2a78be10c",
            sequence: "0xffffffff",
        },
        Input {
            txid: "b7cdd47729936507f8c8efc957a87ea9085214753c62d286f935b36bdc6abcb7",
            vout: 3,
            script_sig: "47304402202ede02a79522a6d64d5088140d7f1abc412c26d9ea093c2fbb321b8f583f279e02207bb12a722558103009abda6f995e857cf04c98c6e8bd01ff7010267bd8a4bd1d0141047146f0e0fcb3139947cf0beb870fe251930ca10d4545793d31033e801b5219abf56c11a3cf3406ca590e4c14b0dab749d20862b3adc4709153c280c2a78be10c",
            sequence: "0xffffffff",
        },
    ],
    outputs: [
        Output {
            amount: 950000,
            script_pub_key: "a91401a2723c1e23a2bfd241c5ded1973e0d1c9ce24887",
        },
        Output {
            amount: 1874250000,
            script_pub_key: "76a9149a803a4f5e37b420a904b1dfcf96c70a7300082d88ac",
        },
        Output {
            amount: 2579368371,
            script_pub_key: "76a91443849383122ebb8a28268a89700c9f723663b5b888ac",
        },
    ],
    locktime: 0,
    transaction_id: "aea1197635c67c73d0e31d4909ddd76f1c3a5be41640035ed06df5c57d20015b",
}
